packages/core/src/common/audioAlertPlayer.ts

```diff
@@ -32,7 +28,7 @@ export class AudioPlayer implements AudioPlayerInterface {
   }
 
   public mute(): void {
-    if (this.player && this.soundID) {
+    if (this.player && (this.soundID || this.soundID == 0)) {
```

Comment:
Can we add a test which validates the soundId = 0 case?

Reply:
Not sure if this is possible since `soundID` is a private parameter that is assigned to a value that Howler returns

Reply:
Ok, making the variable public probably isn't worth it just for this test since the extra public variable could be confusing for customers, so it should be ok without a test.